Submarine Domain Authentication/Domain.Authentication/AuthenticationExceptionMessages.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Diagnosea.Submarine.Domain.Authentication
+{
+    public static class AuthenticationExceptionMessages
+    {
+        private const string Prefix = "Authentication";
+
+        public static readonly string InvalidAudience = $"{Prefix}:InvalidAudience";
```

Comment:
Consider abstracting the `:` to a generic constants file either in Domain.Abstractions or Abstractions.